cmd/support-notifications/res/configuration.toml

```diff
@@ -39,11 +39,10 @@ File = './logs/edgex-support-notifications.log'
   Type = 'mongodb'
 
 [Smtp]
-Host = 'smtp.gmail.com'
-Password = 'mypassword'
-Port = 587
-Sender = 'jdoe@gmail.com'
-Subject = 'EdgeX Notification'
-
-
-
+  Host = 'smtp.gmail.com'
+  Username = 'username@mail.example.com'
+  Password = 'password'
+  Port = 587
+  Sender = 'jdoe@gmail.com'
+  EnableSelfSignedCert = false
+  Subject = 'EdgeX Notification'
```

Comment:
This same modification needs to be made in the `res/docker/configuration.toml`

Reply:
Thanks for noticing, added.